### PR TITLE
Add accessory methods for Codepoint and Address

### PR DESCRIPTION
**Pull request by andreagrandi**

### CODE REVIEW

File: landavailability/api/management/commands/utils.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from api.models import Address, CodePoint
+
+
+def get_address_from_postcode(postcode):
+    return Address.objects.filter(postcode=postcode)
```
Comment (rossjones) [inline landavailability/api/management/commands/utils.py]: Does the input need normalising here somewhere?  Like .replace(' ', '').upper() or something?

File: landavailability/api/management/commands/utils.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+def get_codepoint_from_postcode(postcode):
+    try:
+        return CodePoint.objects.get(postcode=postcode)
```
Comment (rossjones) [inline landavailability/api/management/commands/utils.py]: And here perhaps?

File: landavailability/api/management/commands/utils.py (hunk 1 above)
Comment (andreagrandi) [inline landavailability/api/management/commands/utils.py]: Data in Address and Codepoint is normalised at import time, but input could be different. Good spot. Going to update it

File: landavailability/api/management/commands/utils.py (hunk 2 above)
Comment (andreagrandi) [inline landavailability/api/management/commands/utils.py]: And this too

**Pull request merged**